frontend/src/lib/components/ChildrenFeedback.svelte

```diff
@@ -263,6 +305,11 @@ function printReport() {
 	printWindow.print();
 }
 
+/**
+ * Function for formatting a given datetime string into a more readable format.
+ * @param {string} date - The date to format.
+ * @return {string} The formatted date: day - month - year
+ */
 function formatDate(date: string): string {
```

Comment:
for consistency we should probably use this everywhere in the report, e.g. replace the `new Date().toLocaleDateString()` above